sonic-stormond/scripts/stormond

```diff
@@ -0,0 +1,379 @@
+
+        # If stormond is coming up for the first time, neither STATE_DB info nor JSON file would be present.
+        # In that case, neither resource would have any prior information stored. The baseline is 0 for every field.
+        if self.use_statedb_baseline == False and self.use_fsio_json_baseline == False:
+                fsio_dict["total_fsio_reads"] = fsio_dict["latest_fsio_reads"]
+                fsio_dict["total_fsio_writes"] = fsio_dict["latest_fsio_writes"]
+
+        # If the daemon is re-init-ing after a planned reboot or powercycle, there would be no storage info 
+        # in the STATE_DB. Therefore, we would need to parse the total and hitherto latest procfs reads 
+        # and writes from the FSIO JSON file and use those reads/writes values as a baseline.
+        elif self.use_statedb_baseline == False and self.use_fsio_json_baseline == True:
+            fsio_dict["total_fsio_reads"] = str(int(self.fsio_rw_json[device]["total_fsio_reads"]) + int(fsio_dict["latest_fsio_reads"]))
+            fsio_dict["total_fsio_writes"] = str(int(self.fsio_rw_json[device]["total_fsio_writes"]) + int(fsio_dict["latest_fsio_writes"]))
+
+        # The only scenario where there would be storage info present in the STATE_DB is when the daemon is 
+        # coming back up after a crash.
+
+        # In this scenario, we use the STATE_DB values as the SoT. We use the 'latest_fsio_reads/writes'
+        # values from STATE_DB, which is the values from the last invocation of get_fs_io_reads/writes 
+        # on the storage disk that was posted to STATE_DB, and the values obtained from the most recent
+        # invocation of get_fs_io_reads/writes (prior to this function being called) to determine the 
+        # additional procfs reads and writes that have happened on the FS while the daemon was down. 
+        
+        # We then add these additional values to the previous values of total_fsio_reads/writes to 
+        # determine the new total procfs reads/writes.
+
+        elif self.use_statedb_baseline == True:
+            additional_procfs_reads = int(fsio_dict["latest_fsio_reads"]) - int(self.fsio_rw_statedb[device]["latest_fsio_reads"])
+            additional_procfs_writes = int(fsio_dict["latest_fsio_writes"]) - int(self.fsio_rw_statedb[device]["latest_fsio_writes"])
+
+            fsio_dict["total_fsio_reads"] = str(int(self.fsio_rw_statedb[device]["total_fsio_reads"]) + additional_procfs_reads)
+            fsio_dict["total_fsio_writes"] = str(int(self.fsio_rw_statedb[device]["total_fsio_writes"]) + additional_procfs_writes)
+        
+        return fsio_dict["total_fsio_reads"], fsio_dict["total_fsio_writes"]
+
+
+
+    # Update the Storage device info to State DB
+    def update_storage_info_status_db(self, disk_device, disk_fields, kvp_dict):
+
+        for field in disk_fields:
+            fvp = swsscommon.FieldValuePairs([(field, str(kvp_dict[field]))])
+            self.device_table.set(disk_device, fvp)
```

Comment:
please try using hmset

Reply:
Per our offline discussion, changed the object fvp type and used set which supports setting multiple objects at once.